src/facetgroups/MultiselectDropdownFacetGroup.js

```diff
@@ -50,10 +50,21 @@ const ebiVfSelectStyles = {
   })
 }
 
-const MultiselectDropdownFacetGroup = ({facetGroupName, facets, onChange}) =>
+const tooltipStyle = {
+    background: `white`,
+    border: `none`
+}
+
+const MultiselectDropdownFacetGroup = ({facetGroupName, facetGroupNameDescription, facets, onChange}) =>
   <div className={`padding-bottom-xlarge`}>
-    <h4>{facetGroupName}</h4>
-    <Select components={{ DropdownIndicator, IndicatorSeparator: null }}
+    <h4>{facetGroupName}
+    {facetGroupNameDescription != null &&
```

Comment:
See comments above, and consider them here as well (the HTML in this part is properly nested, though).